src/components/pages/DashboardPage/TotalMembers.js

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from 'react'
+import {connect} from 'react-redux'
+
+import Face from 'material-ui/svg-icons/action/face'
+import InfoBox from './InfoBox'
+import {CircularProgress} from 'material-ui'
+import { updateTotalMembers } from '../../../redux/dashboard/dashboard'
+
+const mapStateToProps = (state) => ({
+  balance: state.get('dashboard').totalMembers.balance,
+  isFetching: state.get('dashboard').totalMembers.isFetching
+})
+
+const mapDispatchToProps = (dispatch) => ({
+  updateTotalMember: () => dispatch(updateTotalMembers())
+})
+
+@connect(mapStateToProps, mapDispatchToProps)
+class TotalMembers extends Component {
+  componentWillMount () {
+    this.props.updateTotalMember()
```

Comment:
typo > updateTotalMember**s**()
but **update()** name would be more than enough